playbook/package.json

```diff
@@ -40,9 +40,11 @@
     "@popperjs/core": "^2.6.0",
     "classnames": "^2.2.6",
     "core-js": "3",
+    "file-loader": "^6.2.0",
     "flatpickr": "^4.6.13",
     "highcharts": "^9.0.0",
     "highcharts-react-official": "^3.1.0",
+    "intl-tel-input": "^17.0.19",
```

Comment:
@carloslimasd Why is this being brought in here, rather than the playbook-website? Are we planning to ship this library with our dist?